plugins/ci/skills/list-symptoms/test_list_symptoms.py

```
@@ -0,0 +1,45 @@
+def test_no_filters_returns_all():
+    assert filter_symptoms(SYMPTOMS) == SYMPTOMS
+
+LABELS = [
+    {"id": "InfraFailure", "label_title": "Infrastructure Failure",
+     "explanation": "Cloud provider or infra problem"},
+    {"id": "ClusterDNSFlake", "label_title": "Cluster DNS Flake",
+     "explanation": "DNS lookups intermittently time out"},
+]
+
+def test_filter_labels_matches_title():
+    assert [l["id"] for l in filter_labels(LABELS, search="dns flake")] == ["ClusterDNSFlake"]
```

Ruff (0.15.21)

[error] Ambiguous variable name: `l`

(E741)